### PR TITLE
docs: add dynaconf NATS HowTo example

### PR DESCRIPTION
**Pull request by sheldygg**
# Description

Please include a summary of the change and specify which issue is being addressed. Additionally, provide relevant motivation and context.

Fixes # (issue number)

## Type of change

Please delete options that are not relevant.

- [x] Documentation (typos, code examples, or any documentation updates)
- [ ] Bug fix (a non-breaking change that resolves an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would disrupt existing functionality)
- [ ] This change requires a documentation update

## Checklist

- [ ] My code adheres to the style guidelines of this project (`scripts/lint.sh` shows no errors)
- [ ] I have conducted a self-review of my own code
- [ ] I have made the necessary changes to the documentation
- [ ] My changes do not generate any new warnings
- [ ] I have added tests to validate the effectiveness of my fix or the functionality of my new feature
- [ ] Both new and existing unit tests pass successfully on my local environment by running `scripts/test-cov.sh`
- [ ] I have ensured that static analysis tests are passing by running `scripts/static-analysis.sh`
- [ ] I have included code examples to illustrate the modifications

### CODE REVIEW

File: docs/docs/en/howto/nats/dynaconf.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+```python linenums="1" hl_lines="7-9"
+from faststream import FastStream
+from faststream.nats import NatsBroker, KvWatch
+
+broker = NatsBroker()
+app = FastStream(broker)
+
+@broker.subscriber("create_sell", kv_watch=KvWatch("order_service"))
+async def watch_kv_order_service(new_value):
```
Comment (Lancetnik) [inline docs/docs/en/howto/nats/dynaconf.md:29]: `new_value: bool` probably?

File: docs/docs/en/howto/nats/dynaconf.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    @broker.subscriber("order_service.order.create.sell")
+    async def handle_create_sell(message: NatsMessage): ...
+    
+    
+    @app.on_startup
+    async def on_startup():
+        await broker.connect()
+    
+        order_service_kv = await broker.key_value("order_service")
+        # await order_service_kv.put("create_sell", b"1")
```
Comment (Lancetnik) [inline docs/docs/en/howto/nats/dynaconf.md:85]: Why it is commented? I don't think, that we should add commented code to the doc

**Comment by Lancetnik** [comment]
@kumaranvpl markup and idea is OK, can you please review and merge it?

**Review by kumaranvpl — APPROVED** [review]

**Comment by kumaranvpl** [comment]
@Lancetnik Proofread and pushed my changes. Will set it to merge.

**Pull request merged**